tests/wallet.rs

```diff
@@ -668,12 +666,11 @@ fn inscribe_gif() {
   let rpc_server = test_bitcoincore_rpc::spawn_with(Network::Regtest, "ord");
   rpc_server.mine_blocks(1)[0].txdata[0].txid();
 
-  let stdout =
-    CommandBuilder::new("--chain regtest --index-sats wallet inscribe --file dolphin.gif")
-      .write("dolphin.gif", [1; 520])
-      .rpc_server(&rpc_server)
-      .stdout_regex("commit\t[[:xdigit:]]{64}\nreveal\t[[:xdigit:]]{64}\n")
-      .run();
+  let stdout = CommandBuilder::new("--chain regtest --index-sats wallet inscribe dolphin.gif")
```

Comment:
```suggestion
  let stdout = CommandBuilder::new("-r --index-sats wallet inscribe dolphin.gif")
```
maybe add to have one integration test testing the short flag

Reply:
I think it's okay not to have an integration test for the short flag, since we have coverage with the unit test.